admin/css/media.css

```diff
@@ -0,0 +1,39 @@
+/*================================= 
+  INI AKTIF KETIKA UKURANNYA 769px keatas 
+==================================== */
```

Comment:
WTH i think you copy paste this css from guru css. yes or no? please remove this if not needed